tools/build-neatvnc.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+# Update and install dependencies
+apt update
+apt install -y meson ninja-build libpixman-1-dev  zlib1g-dev libdrm-dev pkgconf
+
+# Clone and build aml
+git clone https://github.com/any1/aml.git
+cd aml
+meson build
+meson configure build --default-library=static
+ninja -C build
+ninja -C build install
+
+# Clone and build NeatVNC
+cd ..
+git clone https://github.com/any1/neatvnc.git
+cd neatvnc
+git checkout v0.8
+meson build
+meson configure build --default-library=static
+ninja -C build
+ninja -C build install
```

Comment:
At the end of this script, you can show messages to prompt the addtional `PKG_CONFIG_PATH`. See https://people.freedesktop.org/~dbn/pkg-config-guide.html